angularjs/angular.d.ts

```diff
@@ -1414,6 +1417,11 @@ declare module angular {
     * https://docs.angularjs.org/api/ng/service/$http#defaults
     */
     interface IHttpProviderDefaults {
+        /**
+         * {function(data, headersGetter)|Array.<function(data, headersGetter)>}
+         * Transform function or an array of such functions. The transform function takes the http request body and headers and returns its transformed (typically serialized) version.
+         */
+        transformRequest?: Function[]|Function;
```

Comment:
Why is this not a more specific type? It's difficult to understand the pseudo-type you've written in the comment. Please clean this up.